### DISCUSSION TITLE
Compiling index.html content to .js and .css files (Vue 3, Vue-router)

### DISCUSSION THREAD

**Discussion by telefoniero** [vercel/community#757] [Help] [answered]
Hello! I have a problem **after** deployment. My files structure is preserved, but I have my index.html code inside of my **app.css**, **app.js** and **chunk.js** files. Manually building with `npm run build` works fine. What may be the problem? 
File structure is a simple Vue-app dist structure.

**Comment by amyegan** [comment, marked as answer]
_Solution shared in reply https://github.com/vercel/community/discussions/757#discussioncomment-3282245_

Hi @telefoniero. Are you using the `Vue.js` preset in your Build & Development Settings, or are you using custom settings? If you can share your repo or a minimal reproduction of the problem, that will help me get a better idea of what is happening.

**Reply by telefoniero** [reply]
Hi, @amyegan! I forgot to replied. The problem was with the publicPath option in vue.config, I should've used a relative path (./) instead of root path (/), which is default. Pardon me for the late feedback.

**Reply by amyegan** [reply]
No problem. I'm happy you got it working 😁

**Reply by telefoniero** [reply]
Thanks a lot! :)
